client/my-sites/theme/main.jsx

```diff
@@ -863,6 +864,8 @@ class ThemeSheet extends Component {
 				( isExternallyManagedTheme &&
 					( ! isMarketplaceThemeSubscribed || ! isSiteEligibleForManagedExternalThemes ) );
 
+			const noop = () => {};
```

Comment:
We could move this out of the component so it would be declared only once, and not on every re-render. Theoretically, this might save us unnecessary updates and re-renders.